frontend/src/components/Transaction/TransactionForm.js

```diff
@@ -0,0 +1,127 @@
+import React, {Component} from 'react';
+import './TransactionForm.css';
+import TransactionsList from '../TransactionList/TransactionsList';
+
+export default class TransactionForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            account: [],
+            transaction: [],
+            value: '',
+            amount: 0
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleAddDeposit = this.handleAddDeposit.bind(this);
+        this.handleAddWithdrawals = this.handleAddWithdrawals.bind(this);
+    }
+
+    handleAddDeposit() {
+        let today = new Date();
```

Comment:
Looks duplicated in both functions deposit and withdraw. Keep in mind that it's not a good idea client to provide time as it's easy to be manipulated. The server is the one (stub in your case) which should mark the time that operation was performed. The client may pass and time just for verifications, but it shouldn't be meaningful for the server. 

You have several options about this:
1. Extract it into separate function in the component to remove duplication
2. Extract it as separate component 
3. Use timestamp and format date with components if it need to be . 

Keep in mind that it's better server to receive timestamp directly or standard date time format: https://en.wikipedia.org/wiki/ISO_8601